### PR TITLE
Update sidenav order for JS

### PR DESCRIPTION
**Pull request by lizokm**
Updating the order of the sidenav and changing the name of some of the folders, per our conversations with DevRel team :)

### CODE REVIEW

File: docs/platforms/javascript/common/best-practices/index.mdx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Best Practices
-description: "Learn how to set up Sentry in specific scenarios with these best practice guides."
-sidebar_order: 3
+title: Niche Use Cases
+description: "Learn how to set up Sentry for these specific niche scenarios with these best practice guides."
```
Comment (vivianyentran) [inline docs/platforms/javascript/common/best-practices/index.mdx:3]:
```suggestion
description: "Learn how to set up Sentry for specific niche scenarios with these best practice guides."
```

**Review by vivianyentran — APPROVED** [review]

File: docs/platforms/javascript/common/enriching-events/index.mdx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Enriching Events
 description: Add additional context to your events to make them easier to debug.
-sidebar_order: 2300
+sidebar_order: 4
```
[the remark this answers is not in the document]
Comment (lizokm) [reply docs/platforms/javascript/common/enriching-events/index.mdx:4]: Great point.

Comment (lizokm) [inline docs/platforms/javascript/common/enriching-events/index.mdx:4]:
```suggestion
sidebar_order: 5
```

File: docs/platforms/javascript/common/usage/index.mdx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Usage
+title: Capturing Errors and Events
 description: "Learn how to use the SDK to manually capture errors and other events."
-sidebar_order: 10
+sidebar_order: 5
```
Comment (lizokm) [inline docs/platforms/javascript/common/usage/index.mdx:4]:
```suggestion
sidebar_order: 4
```

File: docs/platforms/javascript/common/best-practices/index.mdx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-title: Best Practices
-description: "Learn how to set up Sentry in specific scenarios with these best practice guides."
-sidebar_order: 3
+title: Niche Use Cases
```
Comment (chargome) [inline docs/platforms/javascript/common/best-practices/index.mdx:2]: Not sure if niche use cases is the best title here (especially for non-native speakers).

Some alternatives:

- Special Use Cases
- Special Case Scenarios
- Advanced Use Cases

**Review by chargome — APPROVED** [review]
Thanks for updating these

File: docs/platforms/javascript/common/best-practices/index.mdx (hunk 4 above)
Comment (lizokm) [reply docs/platforms/javascript/common/best-practices/index.mdx:2]: I like Special Use Cases!

Comment (lizokm) [inline docs/platforms/javascript/common/best-practices/index.mdx:2]:
```suggestion
title: Special Use Cases
```

File: docs/platforms/javascript/common/best-practices/index.mdx [hunk 5]
```diff
@@ -1,7 +1,7 @@
...
+description: "Learn how to set up Sentry for specific niche scenarios with these best practice guides."
```
Comment (lizokm) [inline docs/platforms/javascript/common/best-practices/index.mdx:3]:
```suggestion
description: "Learn how to set up Sentry for several specific use cases with these best practice guides."
```

**Pull request merged**